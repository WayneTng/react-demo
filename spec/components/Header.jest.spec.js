import React from 'react'
import renderer from 'react-test-renderer'
import Header from 'components/Header'

describe('<Header />', () => {
  it('renders a tagline', () => {
    const tagline = 'Hello, world!'
    const header = renderer.create(
      <Header tagline={tagline} />
    )

    const tree = header.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
