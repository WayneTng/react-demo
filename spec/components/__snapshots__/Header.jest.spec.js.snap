exports[`<Header /> renders a tagline 1`] = `
<header
  className="top">
  <h1>
    Catch
    <span
      className="ofThe">
      <span
        className="of">
         of 
      </span>
      <span
        className="the">
         the 
      </span>
    </span>
    Day
  </h1>
  <h3
    className="tagline">
    <span>
      Hello, world!
    </span>
  </h3>
</header>
`;
